.versions

```diff
@@ -1,3 +1,3 @@
-ethereum:web3@0.5.0
+ethereum:web3@0.7.0
```

Comment:
it's first time I noticed this file. What it does and why we need it? If it's necessary it might be sensible to add version upgrade to `gulpfile`.